src/Command/EnumCommand.php

```diff
@@ -83,11 +82,10 @@ public function buildOptionParser(ConsoleOptionParser $parser): ConsoleOptionPar
 
         $parser->setDescription(
             'Bake backed enums for use in models.'
-        )->addOption('backing-type', [
-            'help' => 'The return type for the backed enum class',
-            'default' => 'string',
-            'choices' => ['string', 'int'],
-            'short' => 'b',
+        )->addOption('int', [
+            'help' => 'Using backed enums with int instead of string as return type',
+            'boolean' => true,
+            'short' => 'i',
```

Comment:
I don't know if we need a short option for this as `int` isn't that long.

Reply:
Well, it can still be convinient :)

Reply:
Fair enough.